feature/system/src/main/java/io/github/droidkaigi/confsched2020/system/ui/viewmodel/SystemViewModel.kt

```diff
@@ -35,4 +36,20 @@ class SystemViewModel @Inject constructor(
         // launch a Custom Tabs Activity
         customTabsIntent.launchUrl(activity, Uri.parse(url))
     }
+
+
+    fun openCalendar(
+        title: String,
+        location: String,
+        startUnixMillis: Long,
```

Comment:
`DateTime` is preferable to avoid a time-unit mismatch if the class is in the classpath.

Reply:
We should use `DateFormat` , you mean?
I couldn't figure out what I need to change for, sorry.

Reply:
JFYI, the FQDN of `DateTime` is `com.soywiz.klock.DateTime`